requirements.txt

```diff
@@ -1,6 +1,4 @@
-# Django is now defined explicitly in setup.py,
-# so that tox can define it separately from
-# how setup.py includes it.
+Django>=1.11,<2 ; python_version == '2.7'
```

Comment:
It should probably be in a **constraints.txt** file but I'm not sure if that's what you would like. What do you think?